LICENSE

```diff
@@ -272,7 +272,7 @@ That's all there is to it!
 
 --- The Artistic License 1.0 ---
 
-This software is Copyright (c) 2017 by binary.com.
+This software is Copyright (c) 2022 by Binary Services Ltd.
```

Comment:
still binary? not deriv?

Reply:
Binary. It is already released to binary before. One day  binary, forever binary